src/frontends/tests/frontend/shared/test_builtin_extensions_2/CMakeLists.txt

```diff
@@ -4,6 +4,12 @@
 
 set(TARGET_NAME "test_builtin_extensions_2")
 
+if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /ignore:4217")
+    set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} /ignore:4217")
+    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /ignore:4217")
```

Comment:
can we add comments why we disable such warnings? Is it a temporary solution?

Reply:
Yes I will create tickets to solve all warnings.